index.html

```diff
@@ -0,0 +1,164 @@
+                    </div>
+                    <div id="menu-link-cont">
+                    <a href="about.html" class="menu-link">About</a>
+                    <a href="#program" class="menu-link">Program</a>
+                    <a href="#speakers" class="menu-link">Speakers</a>
+                    <a href="#submit-button" class="menu-link">Join</a>
+                    </div>
+                </div>
+            </nav>
+
+
+            <!-- Mobile Navigation Menu (Hidden for Desktop Version) -->
+            <nav id="mobile-navbar" class="navbar-menu">
+                <a id="hamburger-link" href="#" class="toolbar-button"><img id="hamburger" class="hamburger" src="images/icons/mobile-hamburger-icon.png" alt="Open Menu"></a>
+            </nav>
+        </div>
+
+        <div class="header-background" id="header-background">
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Papa_a_la_huanca%C3%ADna" target="_blank"><img class="ingredient-img" src="images/vectors/aji-yellow.png" alt="peruvian ingredient yellow chilli"></a>
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Ceviche" target="_blank"><img class="ingredient-img" src="images/vectors/aji-red.png" alt="peruvian ingredient red chilli"></a>
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Humita" target="_blank"><img class="ingredient-img" src="images/vectors/corn.png" alt="peruvian ingredient corn"></a>
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Pisco_sour" target="_blank"><img class="ingredient-img" src="images/vectors/lemon.png" alt="peruvian ingredient lemon"></a>
+            <a rel="noopener" href="https://es.wikipedia.org/wiki/Archivo:Ensalada_Palta.JPG" target="_blank"><img class="ingredient-img" src="images/vectors/avocado.png" alt="peruvian ingredient avocado"></a>
+            <a rel="noopener" href="https://www.aracari.com/blog/stories/a-guide-to-peruvian-potatoes/" target="_blank"><img class="ingredient-img" src="images/vectors/potatoes.png" alt="peruvian ingredient potatoes"></a>
+        </div> 
+
+
+        <div id="heading-cont" class="heading">
+            <h2 id="main-subtitle" class="main-subtitle">International Culinary Summit</h2>
+            <h1 id="main-title" class="main-title">FLAVORS OF PERU 2022</h1>
+        </div>
+
+        <div id="description-cont">
+            <p id="description" class="description">A closer look at the evolution of Peruvian cuisine, in hand with 5 of the country's most laureated representatives. The ultimate celebration of the senses.</p>
+        </div>
+
+        <div id="dates-venue">
+            <h2 id="dates" class="dates">2022/07/27(WED) - 29(FRI)</h2>
+            <h4 id="venue" class="venue">@ Admiralspalast Berlin, DE</h4>
+        </div>
+
+    </header>
+
+    <section id="program" class="program-container">
+
+        <h2 id="program-main">Main Program</h2>
+
+        <hr class="separator">
+
+        <div id="program-cards-container">
```

Comment:
- [ ] I'm pretty sure the red color and alignment could improve on this part, work on this as a professional application, the recruiters could enter on this application and this could be a good reference for your work as a front-end developer. Kepp this in mind. 🧠 
![image](https://user-images.githubusercontent.com/20563252/137224484-c1883039-682d-48b9-8066-c9bb3eb6a886.png)